Web/Admin.UI/src/app/components/tenant/facility-view/facility-view.component.html

```
@@ -3,25 +3,27 @@
     <div class="link-card-header">
       <div class="split-container" style="margin-bottom: 0;">
             <div class="left-pane">
-               <p class="title">{{ facilityConfig?.facilityName }}
+               <p class="title" data-testid="facility-view-title">{{ facilityConfig?.facilityName }}
               <p>
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </p></p> ], start tag match failed [ <p> ] on line 7.

(tag-pair)